src/components/Header/Header.js

```diff
@@ -1,12 +1,31 @@
-import React from 'react';
+import React, { Component } from 'react';
 import HeaderNav from './HeaderNav/HeaderNav';
+import HeaderUserMenu from './HeaderUserMenu/HeaderUserMenu';
+import { ReactComponent as DopplerLogo } from '../../doppler-logo.svg';
+
+class Header extends Component {
+
+  constructor() {
+    super();
+    this.state = {
+      user: {},
+    };
+  }
+
+  componentWillMount() {
+    this.setState({ user: this.props.user })
```

Comment:
If I am not wrong, when something is not changed by the component, it should be dealt as _props_ and not as _state_, what do you think?

Reply:
Yes! this it's not final. Need the real data and pass only what child needs. I need change this, i was trying things

Reply:
Great!